### DISCUSSION TITLE
Button classes to be configurable

### DISCUSSION THREAD

**Discussion by vlados** [wireui/wireui#91] [Ideas]
Right now the button classes are hardcoded in [Button.php](https://github.com/PH7-Jack/wireui/blob/edc0f60d43969787d5067813ac5d1a637d4895b6/src/View/Components/Button.php#L5)
I think it's better this classes (and not only for the button) to be configurable, or to be able to create a "theme" for the current project

**Comment by ph7jack** [comment]
@vlados now, in the v0.13.0 release, is possible to extend all components, just extends the button component and add your custom colors/classes
